src/modules/Exposure/info.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Exposure",
+  "description": "Change the exposure of the image by given exposure value",
+  "inputs": {
+      "exposure": {
+          "type": "range",
+          "desc": "exposure value for the new image",
+          "default": "0",
+          "min": "-4",
+          "max": "4",
```

Comment:
How about using a percentage and dividing by 100? This might help the user put exact values as the input type can be changed to number from range

Reply:
100% exposure wouldn't be intuitive I think. Think about it, if someone wants a higher exposure, a '100%' exposure doesn't really make sense. Like brightness itself, a graduated scale gives a better idea of the increasing nature of the exposure factor.